### DISCUSSION TITLE
GitHub isn't working well under Visual Studio 2022

### DISCUSSION THREAD

**Discussion by Xandones** [community/community#66702] [Other Feature Feedback, Questions, & Ideas]
### Topic Area

Product Feedback, Visual Studio 2022, GitHub

### Body

**I can create a new project on GitHub under Visual Studio Community 2022** but **I can't update it using push.** 
Everything looks okay on Visual Studio (it shows that it worked but when I checked on GitHub's page it didn't).

Can you help me?

**Comment by yeabtsegataye** [comment]
it work know

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Xandones ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Xandones** [reply]
Thank you. I hope this helps me get an answer quicker.
